tests/components/fritzbox/test_climate.py

```diff
@@ -68,8 +70,8 @@ async def test_setup(hass: HomeAssistant, fritz: Mock) -> None:
     assert state.attributes[ATTR_PRESET_MODE] is None
     assert state.attributes[ATTR_PRESET_MODES] == [PRESET_ECO, PRESET_COMFORT]
     assert state.attributes[ATTR_STATE_BATTERY_LOW] is True
-    assert state.attributes[ATTR_STATE_HOLIDAY_MODE] == "fake_holiday"
-    assert state.attributes[ATTR_STATE_SUMMER_MODE] == "fake_summer"
+    assert not state.attributes[ATTR_STATE_HOLIDAY_MODE]
+    assert not state.attributes[ATTR_STATE_SUMMER_MODE]
```

Comment:
Could we use `is False` instead as I read it initially as the attributes was removed while they are not